CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ can and probably will change functionality and break backwards compatability
 at anytime.
 
 ## [Unreleased]
+### Fixed
+ * Catch UnknownNameErrors that don't need to be thrown. #479
```

Comment:
damn.  I was hoping github's markdown would recognize and link to this automatically.  It doesn't 😢